### DISCUSSION TITLE
How do I  read rtmp stream nobuffer

### DISCUSSION THREAD

**Discussion by xiaozhubenben** [PyAV-Org/PyAV#927] [1. Help]
## Overview
Hi,
    I'm using pyav read from rtmp stream.  Using ffmpeg commandline , i can us fflag nobuffer to reduce  latency, but I'am new to pyAV , I dont know how to write nobuffer param to options, so please show me an example for this ,thanks.

I have done the following:

- [x ] Checked the [PyAV documentation](https://pyav.org/docs)
- [x] Searched on [Google](https://www.google.com/search?q=pyav+how+do+I+foo)
- [x] Searched on [Stack Overflow](https://stackoverflow.com/search?q=pyav)
- [x ] Looked through [old GitHub issues](https://github.com/PyAV-Org/PyAV/issues?&q=is%3Aissue)
- [x] Asked on [PyAV Gitter](https://gitter.im/PyAV-Org)
- [ ] ... and waited 72 hours for a response.

**Comment by SoundsSerious** [comment]
Would love to know this as well!

**Comment by Dian-Yi** [comment]
you can set ffmpeg `analyzeduration` to reduce latency when you using` av.open` open RTSP stream
